soccer/gameplay/plays/sequences/move_sequence.py

```diff
@@ -0,0 +1,25 @@
+import single_robot_sequence
+import role_assignment
+import skills.move
+import behavior
+
+#Moves a SINGLE robot in a sequence of points
+class MoveSequence(single_robot_sequence.SingleRobotSequence):
```

Comment:
You may be able to just use SingleRobotSequence directly instead of having a subclass specialized for Move. Then you could use it like:

~~~{.py}
seq = SingleRobotSequence(behaviors = [skills.move.Move(pos) for pos in my_positions])
~~~